AudioAnalytics/AudioAnalyzer/Program.cs

```diff
@@ -27,12 +27,21 @@ class Program
         private const string OutputFolderName = @"Output";
 
         /// <summary>
-        /// The main method of the sample. Please make sure you have set settings in appsettings.json
+        /// The main method of the sample. Please make sure you have set settings in appsettings.json or in the .env file in the root folder
         /// </summary>
         /// <param name="args"></param>
         /// <returns></returns>
         public static async Task Main(string[] args)
         {
+            try
+            {
+                DotEnv.Load(Path.Combine(Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.Parent.Parent.Parent.FullName, ".env"));
```

Comment:
add Comments here on what is going on and how to use the .env. 
May note that the sample.env needs to be filled out and renamed.